### PR TITLE
[Plots] Allow user to change marker style

### PR DESCRIPTION
**Pull request by davetsay**
## Overview
Addresses #3134
- Add circle marker style

## Testing Instructions 
see #3134

## Author Checklist
| | |
| --- | :---: |
| Changes address original issue? | Y |
| Unit tests included and/or updated with changes? | N (won't do for plots) |
| Command line build passes? | Y |
| Changes have been smoke-tested? | Y |
| Testing instructions included? | Y |

### CODE REVIEW

**Comment by davetsay** [comment]
@charlesh88 , we already discussed UI stuff but included you just in case. I'll probably submit a separate UI PR later.
@JoelMcKinnon , since you like to keep abreast with current plot affairs.
@deeptailor , vista.

File: src/plugins/plot/res/templates/plot-options-browse.html [hunk 1]
```diff
@@ -56,7 +56,7 @@ <h2 title="Plot series display properties in this object">Plot Series</h2>
                     <div class="grid-cell label"
                          title="Whether markers are displayed, and their size.">Markers</div>
                     <div class="grid-cell value">
-                        {{series.get('markers') ? "Enabled: " + series.get('markerSize') + "px" : "Disabled"}}
+                        {{ series.get('markers') ? series.get('markerShape')[0].toUpperCase() + series.get('markerShape').substring(1) + ": " + series.get('markerSize') + "px" : "Disabled" }}
```
Comment (JoelMcKinnon) [inline src/plugins/plot/res/templates/plot-options-browse.html:59]: I see this is one of those cases where a ternary is useful, but it does challenge readability. Could this be a computed property in which a standard if condition was used?

Comment (davetsay) [reply src/plugins/plot/res/templates/plot-options-browse.html:59]: Unfortunately, this is an angular template. If it were a Vue SFC I'd totally agree.

Comment (deeptailor) [reply src/plugins/plot/res/templates/plot-options-browse.html:59]: could we have a helper method to get us what we need here?

File: src/plugins/plot/src/draw/Shapes.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            drawC2D: function (x, y, size) {
+                const radius = size / 2;
+
+                this.c2d.beginPath();
+                this.c2d.arc(x, y, radius, 0, 2 * Math.PI, false);
+                this.c2d.closePath();
+                this.c2d.fill();
+            }
+        }
+        // cross: {
```
Comment (deeptailor) [inline src/plugins/plot/src/draw/Shapes.js:50]: Remove commented code

File: src/plugins/plot/res/templates/plot-options-browse.html (hunk 1 above)
Comment (davetsay) [reply src/plugins/plot/res/templates/plot-options-browse.html]: Alright. The refactors have it.

Moved the logic into a function in series.

File: src/plugins/plot/src/draw/Shapes.js (hunk 2 above)
Comment (davetsay) [reply src/plugins/plot/src/draw/Shapes.js]: done

**Comment by davetsay** [comment]
@charlesh88 , @deeptailor , @JoelMcKinnon , addressed reviewer comments and added new shapes (circle, diamond, triangle). Ready for re-review.

**Comment by davetsay** [comment]
@deeptailor please merge when possible as you've requested changes

**Comment by deeptailor** [comment]
## Reviewer Checklist
Changes appear to address issue? Yes
Appropriate unit tests included? N/A
Code style and in-line documentation are appropriate? Yes
Commit messages meet standards? Yes
Has associated issue been labelled unverified? (only applicable if this PR closes the issue) Yes

**Pull request merged**